classes/mail/MailTemplate.inc.php

```diff
@@ -97,47 +77,19 @@ function MailTemplate($emailKey = null, $locale = null, $enableAttachments = nul
 		}
 
 		$userSig = '';
+		$user = $request->getUser();
 		if ($user && $this->includeSignature) {
 			$userSig = $user->getLocalizedSignature();
 			if (!empty($userSig)) $userSig = "<br/>" . $userSig;
 		}
 
-		if (isset($emailTemplate) && $request->getUserVar('subject')==null && $request->getUserVar('body')==null) {
+		if (isset($emailTemplate)) {
 			$this->setSubject($emailTemplate->getSubject());
 			$this->setBody(nl2br($emailTemplate->getBody() . $userSig));
 			$this->enabled = $emailTemplate->getEnabled();
-
-			if ($request->getUserVar('usePostedAddresses')) {
-				$to = $request->getUserVar('to');
-				if (is_array($to)) {
-					$this->setRecipients($this->processAddresses ($this->getRecipients(), $to));
-				}
-				$cc = $request->getUserVar('cc');
-				if (is_array($cc)) {
-					$this->setCcs($this->processAddresses ($this->getCcs(), $cc));
-				}
-				$bcc = $request->getUserVar('bcc');
-				if (is_array($bcc)) {
-					$this->setBccs($this->processAddresses ($this->getBccs(), $bcc));
-				}
-			}
```

Comment:
I did a bunch of cleanup of the old OJS 2.x emailing stuff. We used to re-post the email composition form multiple times to do things, and refer to the form from various parts of the system using URL parameters, etc., and those are no longer used and a bad idea.